odl/tomo/analytic/filtered_back_projection.py

```diff
@@ -50,27 +196,44 @@ def fbp_op(ray_trafo, padding=True):
         `CircularConeFlatGeometry` : Approximate reconstruction, correct in
         limit of fan angle = 0 and cone angle = 0.
 
-        `HelicalConeFlatGeometry` : Very approximate.
+        `HelicalConeFlatGeometry` : Very approximate unless a
+        `tam_danielson_window` is used. Accurate with the window.
```

Comment:
I did some more thinking about this, I realized that adding more discretization points will **not** allow this cheap trick to converge to the correct solution. Even in the continuum limit, this method is inexact (but still rather good).

Anyway, as you see from the example the error is quite small.

Regarding noise, that is actually the main reason people are still doing something like this still in the clinic. The Katsevich method is even more sensitive to noise, and especially to geometry errors.